building/target_platforms/target_platforms_updates.adoc

```diff
@@ -71,7 +71,11 @@ Provide enough detail to explain why the change is needed, and what is required
 
 * _Original p2 repo_: which repository are the plugins/features from? eg., http://dl.google.com/eclipse/plugin/4.2
 
-* _JBoss mirror_: mirror URL to use, eg. http://downloads.jboss.org/jbosstools/updates/requirements/gwt/3.1.0.v201208080121-rel-r42/
+* _JBoss mirror_: mirror URL to use, eg. http://downloads.jboss.org/jbosstools/updates/requirements/tern/0.7.0.201412012159/
+
+** If no mirror yet exists, look in https://github.com/nickboldt/jbosstools-download.jboss.org/tree/master/jbosstools/updates/requirements[updates/requirements/] for your requirement's folder. In that folder should be a build.xml file showing the URL and version to mirror. Please submit a pull request to update the URL or version, as applicable. 
+** If no build.xml file exists for your requirement, please submit a pull request to create one.
```

Comment:
Because the last time I tried it, it didn't support IU filtering or removal of references to eclipse.org update sites from the resulting site's metadata. And the script was MUCH longer than the simple ant one. I know you hate Ant (simply because you do) but this works.

Here's what the maven equivalent of the ant script looked like 3 years ago:

http://download.jboss.org/jbosstools/updates/requirements/m2eclipse/pom.xml

Here's what the latest version of a requirements mirror script looks like today:

http://download.jboss.org/jbosstools/updates/requirements/thym/build.xml